packages/dbml-core/src/connector/index.js

```diff
@@ -0,0 +1,13 @@
+import DatabaseConnector from '../parse/DatabaseConnector';
+import ModelExporter from '../export/ModelExporter';
+
+async function _fetch (connection, format) {
```

Comment:
Are you adding _fetch to avoid collision with the global [fetch function in node 18](https://nodejs.org/dist/latest-v18.x/docs/api/globals.html#fetch)? If yes then I don't understand why you are re-exporting it as `fetch`. Have you consider a different name?